src/Pages/Categories.jsx

```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>Check status</h1>
```

Comment:
The categories component should display a button with text `Check Status`. However, you are displaying a header tag, kindly check and fix this. :wink:

Reply:
- Well resolved @ivanmvh 👍🏽 
<img width="1098" alt="Screenshot 2023-01-20 at 16 46 12" src="https://user-images.githubusercontent.com/58692590/213742702-3b6004e6-c5cc-42dd-b582-a0f4f1b4e09e.png">